devices/osram.js

```diff
@@ -4,6 +4,27 @@ const ota = require('../lib/ota');
 const reporting = require('../lib/reporting');
 const extend = require('../lib/extend');
 const e = exposes.presets;
+const utils = require('../lib/utils');
+
+const fzLocal = {
```

Comment:
@Koenkk - I am not particularly happy with this.

To do this 'properly' in my head, I need to add another item in the `batteryVoltageToPercentage` section of `utils.js` and I need add all of the `batteryAlarmState`'s as per 3.3.2.2.4.11 in the Zigbee Cluster Library specification.

For now, it seemed simpler to just drop it into it's own local converter...

Reply:
> I need to add another item in the batteryVoltageToPercentage section of utils.js

Why not do this? 

>  need to refactor and add all of the batteryAlarmState's as per 3.3.2.2.4.11 in the Zigbee Cluster Library specification.

For the battery low, we take `1<<3` into account here:

 https://github.com/Koenkk/zigbee-herdsman-converters/blob/c35dfceeb1833f77fd9aa3f9146395ae96f25ef9/converters/fromZigbee.js#L344

Reply:
Happy to add the new batteryVoltageToPercentage value.

As for the batteryAlarmState, I don't think we do..? Correct me if I'm wrong, but arent we just taking the bitshifts of the first threshold values for battery 1, 2 and 3, but not the second, third or fourth as in the ZCL spec - this device uses the fourth four some unknown reason...

Reply:
We can take all here (so add a `battery4Low` to the expression)

Reply:
Done and ready for full review :-)